onboarding.md

```diff
@@ -208,7 +208,6 @@ needs to be pointed out separately during the onboarding.
 * Run CI on the PR. Use the `node-test-pull-request` CI task.
 * After two Collaborator approvals for the change and two Collaborator approvals
   for fast-tracking, land the PR.
-* Leave a comment in the PR: `Please 👍 this comment to approve fast-tracking`.
 * If there are not enough approvals within a reasonable time, consider the
   single approval of the onboarding TSC member sufficient, and land the PR.
```

Comment:
I think we should keep mentioning the comment, at least in the previous step (it's a bit unclear what is an "approval for fast-tracking" if there's no mention of the comment). Also, should we add a step in case GH Actions are down and the comment doesn't come in time?

Reply:
I added mention of the GitHub bot comment in the bullet point that instructs the collaborator-to-be to add the `fast-track` label. I can add additional text asking them to leave a comment if the bot doesn't, but I think there's a downside to elaborating on everything that might go wrong, and this is one of those cases where the thing to do is hopefully obvious, especially to the TSC member doing the onboarding: Post the comment yourself and/or report the failure-to-comment to the keepers of the bot. And, of course, if this ends up being a frequent problem during onboardings, we can add that text later.

All that said, I don't feel *that* strongly about it and will add that text if it's the difference between getting a review approval from you and not. 

PTAL.